docs/chat_setup.md

```diff
@@ -1,5 +1,37 @@
 # Installing Nautobot Chatops
 
+## Installation
+
+The plugin is available as a Python package in PyPI and can be installed with `pip3` after logging in with the `nautobot` user account.
+
+```shell
+sudo -iu nautobot
+pip3 install nautobot-chatops
+```
+
+> The plugin is compatible with Nautobot 1.0.0beta1 and higher
+
+Once installed, the plugin needs to be enabled in your `nautobot_config.py`
+
+```python
+# In your nautobot_config.py
+PLUGINS = ["nautobot_chatops"]
+
+PLUGINS_CONFIG = {
+    "nautobot_chatops": {
+         #     ADD YOUR SETTINGS HERE
+    }
+}
+```
+
+Nautobot supports `Slack`, `MS Teams` and `Webex Teams` as backends but by default all of them are disabled. You need to explicitly enable the chat platform(s) that you want to use in the `PLUGINS_CONFIG` with one or more of `enable_slack`, `enable_ms_teams` or `enable_webex_teams`.
+
+The plugin behavior can be controlled with the following list of general settings:
+
+| Configuration Setting        | Description | Mandatory? | Default |
+| ---------------------------- | ----------- | ---------- | ------- |
+| `delete_input_on_submission` | After prompting the user for additional inputs, delete the input prompt from the chat history | No | `False` |
+
```

Comment:
I'd like to see this left where it was on the main page, keeping the plugin documentation consistent on the install instructions personally. I'm also ok if this moves to here.

This is something else that I'm noticing as part of this PR is that `Mattermost` is missing from the list of backends, which is now supported.

Reply:
I think we have enough present. My original comment was to keep install locally in the main readme. My mind is changed on that front!

Reply:
Can we have the author of the Mattermost change update the documentation appropriately?  I have never used the platform and know nothing of the setup.

Reply:
I'll double check on the setup if it is there. But the front page on line 27 of the readme should have that update.

Reply:
Oh, I see what you are saying now.